src/common/components/ui/formik/InputField.tsx

```diff
@@ -24,7 +32,7 @@ const InputField: React.FC<
       inputError,
       description,
       labelInfo,
-      showErrorMessage = true,
+      // showErrorMessage = true,
```

Comment:
there is couple of places where this prop is still used. I think if we change
```
{errorMessage && (
  <HelperText fieldId={fieldId} isError>
    {errorMessage}
  </HelperText>
)}
```

to
```
{errorMessage && showErrorMessage && (
   <HelperText fieldId={fieldId} isError>
     {errorMessage}
   </HelperText>
)}
```
we should be good